compress.cc

```diff
@@ -172,7 +175,7 @@ std::map<sstring, sstring> compression_parameters::get_options() const {
     }
     auto opts = _compressor->options();
 
-    opts.emplace(compression_parameters::SSTABLE_COMPRESSION, _compressor->name());
+    opts.emplace(compression_parameters::CLASS, _compressor->name());
```

Comment:
This is part of schema mutations ABI, exchanged between nodes. I think this change can cause schema propagation failures in a mixed cluster during rolling upgrade.

Reply:
Ouch. I'll verify and add backward compatibility code if needed.

Reply:
I have another idea. Retain the user's spelling everywhere, and choose at the last moment (when instantiating the compressor). We can keep the dual spelling for the usual two years, then switch the spelling to the new one uncondionally.